include/OpenSpaceToolkit/Astrodynamics/Flight/Profile.hpp

```diff
@@ -292,7 +292,7 @@ class Profile
     /// @param anOrbit An orbit
     /// @param anOrbitalFrameType An orbital frame type
     /// @return Flight profile
-    static Profile NadirPointing(
+    static Profile LocalOrbitalFrame(
         const trajectory::Orbit& anOrbit, const trajectory::Orbit::FrameType& anOrbitalFrameType
```

Comment:
For clarity could you also rename this in Orbit to `LocalFrameType`?

Reply:
Rename the enum? I can do that in a separate MR, that will be a pretty big change to apply across repos...

Reply:
Will do this in a separate MR.